utils/sendGraphQLRequest.ts

```diff
@@ -0,0 +1,56 @@
+import { GraphQLClient } from 'graphql-request';
+import config from './config';
+
+const endpoint = config.apiUrl;
+const graphQLClient = new GraphQLClient(endpoint, {
+  headers: {
+    authorization: `Bearer ${config.apiToken}`,
+    'Content-Type': 'application/json',
+  },
+});
+
+/**
+ * log errors from DS
+ * @param {*} apiResponse 
+ */
+const logErrorsFromDS = (apiResponse: {response: {errors: string[]}}): void => {
+  if (apiResponse.response) {
+    if (apiResponse.response.errors.length > 0) {
+      apiResponse.response.errors.forEach((err) => {
+        console.error('Error details', err);
+      });
+    }
+  }
+};
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Shouldn't we use DSReponse here and remove this comment (and the others below)?

Reply:
no because we have other queries (like GroupeInstructeur)